.prettierrc

```diff
@@ -3,5 +3,18 @@
   "printWidth": 80,
   "singleQuote": true,
   "trailingComma": "all",
-  "useTabs": false
+  "useTabs": false,
+  "overrides": [
+    {
+      "files": "*.sol",
+      "options": {
+        "printWidth": 80,
+        "tabWidth": 4,
+        "useTabs": false,
+        "singleQuote": false,
+        "bracketSpacing": false,
+        "explicitTypes": "always"
+      }
+    }
+  ]
```

Comment:
we should not have to specify this as the plugin does exactly this

Reply:
can't remove it it seems...